docs/core/tools/dotnet-publish.md

```diff
@@ -139,28 +138,6 @@ For more information, see the following resources:
 
     If you specify a relative path when publishing a solution, each project's output goes into a separate folder relative to the project file location. If you specify an absolute path when publishing a solution, all publish output for all projects goes into the specified folder.
 
-- **`-p:PublishReadyToRun=true`**
```

Comment:
Are we concerned that removing these options here decreases the chances of people finding them and learning about what they do? I think it is discoverable to list these publish options in the "dotnet publish" document, so people discover them.

Reply:
I'm slightly concerned, but removed it based on the discussion in https://github.com/dotnet/docs/pull/24740#discussion_r654626080. We do have dedicated documentation for these properties. Maybe the best thing would be to link to the important publish MSBuild options from the publish options doc, without mentioning them as command-line arguments? cc @agocke

Reply:
> link to the important publish MSBuild options from the publish options doc, without mentioning them as command-line arguments?

I like that approach. That way they are still discoverable.